06-building-a-program-start-to-finish/writing-tests/output/invertedSpec.js

```diff
@@ -0,0 +1,152 @@
+const validBook = require('./test/testFiles/validBook.json');  //  book with valid contents
```

Comment:
You have multiple cases of eslint "_**exceeds the maximum line length of 80**_" violations in this test file. Please fix them all